README.md

```diff
@@ -1,13 +1,103 @@
-# Automated Surveys with Twilio
+<a href="https://www.twilio.com">
```

Comment:
Please take a look to the [docs guidelines](https://github.com/twilio/docs-guidelines/blob/fb01ef506476a9d9694cddc22721bd0c7a88c11e/templates/README/node/express/README.md). Not sure why we are adding the step for installing a dependency like mongo, wouldn't be enough just to indicate that one of the dependencies of this application is Mongo DB?

I asked @hortega to help out updating that README. Could you guys work on that chore together. I'll add a chore in PT for addressing this request. If you have any questions, please let me know.